### DISCUSSION TITLE
关于alt+滚轮调节系统音量

### DISCUSSION THREAD

**Discussion by 6Tom** [shuax/MouseInc.Settings#269] [Ideas]
大佬佬，关于alt+滚轮调节系统音量这项功能，可以放开键位设置吗？

比如我的畅想，能否改为win+alt+滚轮，这样比如photoshop、indesign等就完全不会冲突了，既不用排除这些软件，也不用来回暂停/启用

我不了解这项工作是否会有很大工作量，所以，我提出来主要是作为畅想，看您时间精力和意愿

祝大佬佬生活愉快，期待回复

**Comment by shuax** [comment]
改倒是不麻烦，但是这个不是所有人都需要的。

**Reply by 6Tom** [reply]
是的，不过对于我这类从事设计行业的同志们来说，我觉得这个想法，是可以带来难以数尽的诸多便利

谢谢大佬佬，但是还是看您意愿，如果能做出来可以说是感激不尽

或者说如果您有这个意愿，我也愿意有偿提议

**Comment by shuax** [comment]
[MouseInc.zip](https://github.com/shuax/MouseInc.Settings/files/8620353/MouseInc.zip)
临时改的，你可以自己拿去用。

**Reply by 6Tom** [reply]
妈呀，谢谢大佬佬，真是万分感谢，提供个捐助的途径吧

**Reply by shuax** [reply]
> 妈呀，谢谢大佬佬，真是万分感谢，提供个捐助的途径吧

算了，基金都亏麻了，不差这点钱。

**Reply by 6Tom** [reply]
再次感谢，对于设计人员来讲这意义很大，大佬佬生活愉快

**Reply by 6Tom** [reply]
对了大佬佬，我发现这样还有个优点，这样就解放了按住alt+tab界面的中键单击关闭作用，就不非得因为忘记，按住alt+tab之后还要换到win+tab，或者alt+ctrl+tab，或者alt+ctrl+shift+tab，或者边缘滚动了

**Reply by odyssey8848** [reply]
大佬，定制版好像屏蔽了alt+滚轮，没有找到可以自定义设置的地方

**Reply by 6Tom** [reply]
因为不是自定义的，是win+alt+滚轮

**Reply by 6Tom** [reply]
> 大佬，定制版好像屏蔽了alt+滚轮，没有找到可以自定义设置的地方

兄弟，我自己找了一些新办法，我觉得你可能跟我是同行，就想着一定要回来分享给你，看看我刚写的这个链接下面的
[https://github.com/shuax/MouseInc.Settings/discussions/323](url)
